extensions/amp-analytics/amp-analytics.md

```diff
@@ -490,6 +490,23 @@ Use this configuration to fire a request when the page becomes is hidden.  The f
 }
 ```
 
+A [`visibilitySpec`](#visibility-spec) can be included so that a request is only fired if the visibility duration conditions are satisfied.
```

Comment:
I intentionally called out "duration condition" because other condition like `visiblePercentageMin` is simply ignored. 

Or do you think I should make it more explicit about the fact that `visiblePercentageMin` is ignored?